### PR TITLE
qa: checkstyle, pmd

### PR DESCRIPTION
**Pull request by soloturn**
addressed pmd and checkstyle warnings:
* line too long
* logger not parameterized
* intellij warnings about formatting, e.g. @ReceiveEvent(components = {InventoryComponent.class}) replaced with  @ReceiveEvent(components = InventoryComponent.class)
* unused private field

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ build.gradle
 bin/
 
 build/
+/src/test/resources
```
Comment (jdrueckert) [inline .gitignore]: I'm thinking maybe ignoring *.xml makes more sense here?

File: src/main/java/org/terasology/module/inventory/systems/InventoryUtils.java [hunk 2]
```diff
@@ -237,7 +235,8 @@ private static boolean validateMoveAmount(EntityRef instigator, EntityRef from,
             return false;
         }
 
-        BeforeItemRemovedFromInventory removeFrom = new BeforeItemRemovedFromInventory(instigator, InventoryUtils.getItemAt(from, slotFrom), slotFrom);
+        BeforeItemRemovedFromInventory removeFrom = new BeforeItemRemovedFromInventory(instigator, InventoryUtils.getItemAt(from,
+                slotFrom), slotFrom);
```
Comment (jdrueckert) [inline src/main/java/org/terasology/module/inventory/systems/InventoryUtils.java]: Consider breaking lines at more readable points, e.g.
```
        BeforeItemRemovedFromInventory removeFrom = new BeforeItemRemovedFromInventory(instigator,
                InventoryUtils.getItemAt(from, slotFrom), slotFrom);
```

File: src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java [hunk 3]
```diff
@@ -39,8 +39,8 @@ public class InventoryGrid extends CoreWidget {
     /**
      * Defines the first inventory slot index to which an InventoryGrid should refer to.
      *
-     * Example: If an inventory has 20 slots and cellOffset is set to 5, 15 cells will be drawn starting from slot no. 5 (starting from zero).
-     * Other slots will be not accessible by this InventoryGrid.
+     * Example: If an inventory has 20 slots and cellOffset is set to 5, 15 cells will be drawn starting from slot no. 5 (starting from
+     * zero). Other slots will be not accessible by this InventoryGrid.
```
Comment (jdrueckert) [inline src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java:43]: Consider breaking lines at more readable places, e.g.
```
     * Example: If an inventory has 20 slots and cellOffset is set to 5, 15 cells will be drawn 
     * starting from slot no. 5 (starting from zero). Other slots will be not accessible by this InventoryGrid.
```

**Review by jdrueckert — APPROVED** [review]

**Comment by jdrueckert** [comment]
While conventional commits are not mandatory within a PR (due to the fact that we're typically squash-merging), it does help people merging your PRs if you at least follow the convention in your PR titles

File: src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java (hunk 3 above)
Comment (soloturn) [reply src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java:43]: this is standard text, and i do love the line breaks at the end of the line, not arbitrary positions.

**Comment by soloturn** [comment]
> While conventional commits are not mandatory within a PR (due to the fact that we're typically squash-merging), it does help people merging your PRs if you at least follow the convention in your PR titles

rebased on top of your pull request so your commits with the respective commit messages are there.

File: src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java (hunk 3 above)
Comment (jdrueckert) [reply src/main/java/org/terasology/module/inventory/ui/InventoryGrid.java:43]: it's not arbitrary but based on read flow :wink:

**Comment by jdrueckert** [comment]
can you please include in the PR description, which checkstyle and pmd warnings you addressed?
for one you already did it in your commit message (checkstyle, remove empty javadoc), but for the remaining changes it's unclear

**Comment by soloturn** [comment]
> can you please include in the PR description, which checkstyle and pmd warnings you addressed? for one you already did it in your commit message (checkstyle, remove empty javadoc), but for the remaining changes it's unclear

updated pull request text, as well as commit message of the respective commit accordingly.

**Pull request merged**